wormhole/module.py

```diff
@@ -92,6 +92,14 @@ async def _message_formatter(self, message: discord.Message) -> str:
         formatted_message = (
             f"**{guild_display} {message.author.name}:** {message.content}"
         )
+
+        if message.reference and message.reference.type == MessageReferenceType.reply:
+            msg = message.reference.cached_message.content.replace("\n", "\n> ")
```

Comment:
I see only one potential problem - what if thecached_message or content is empty? Not sure if that might happen tho.